git/git_test.go

```diff
@@ -0,0 +1,304 @@
+/*
+Copyright 2021 The Flux authors
```

Comment:
Across the new files:

```suggestion
Copyright 2022 The Flux authors
```